.codecov.yml

```diff
@@ -0,0 +1,4 @@
+fixes:
+ # Ensure Codecov can follow files within symlinked directories
```

Comment:
There are a few symlink files. Do we need to list them as well?

Reply:
It's able to follow symlinked files ([it sees routes.rb fine](https://codecov.io/gh/chanzuckerberg/sorbet-rails/src/57b8621503f45920a3fcfb60be70416bb9f4c04b/spec/support/5.1.7/config/routes.rb)), just not files within symlinked directories. It's also not entirely clear that this will fix it, but it _should_.